README.md

```diff
@@ -0,0 +1,138 @@
+
+A Fermat cofactor is the integer $F_n / P$, where $P$ is the product of all known factors of $F_n$.
+
+### Pépin's test
+Pépin's test is a definitive primality test that can be used to determine if a Fermat number is prime. If $F_n$ is the $n^{th}$ Fermat number, then $F_n$ is prime if and only if $3^{(F_n - 1)/2} \equiv -1$ (mod $F$). The term $3^{(F_n - 1)/2}$ (mod $F$) may be calculated by $2^n-1$ squarings of the base $3$, modulo $F_n$. Pépin’s test is also a definitive primality test when performed with bases other than 3, such as 5, 6, and so on.
+
+### Suyama cofactor test
+The Suyama test utilizes Fermat’s little theorem to test a Fermat number's cofactor for probable primality.
+It was devised by Hiromi Suyama in 1984 and subsequently improved upon by Hendrik W. Lenstra, Jr.
+If a Fermat number $F_n$ is composite and equal to $P \times C$, where $P$ is the product of known factors and $C$ is the cofactor, then the Suyama test is performed as follows:
+
+* Calculate $A = b^{F_n-1}$ (mod $F_n$) for some base $b$. If base = 3 then this is the square mod $F_n$ of the final Pepin residue.
+* Calculate $B = b^{P-1}$ (mod $F_n$).
+* Calculate $R = (A - B)$ (mod $C$). If $R = 0$ then the cofactor $C$ is a probable prime; otherwise it is composite.
+
+Proving the cofactor composite requires finding only one base for which $R \neq 0$. cofact (main branch) only supports base $b = 3$ for the Suyama test. So far, this has been sufficient since each Fermat cofactor from $F_{12}$ through $F_{30}$ is currently composite.
+
+Finally, the cofactor $C$ can be tested to determine if it is a prime power by calculating the greatest common divisor $G = \text{gcd}(A - B, C)$. If $G = 1$ then the cofactor is not a prime power. If $G \neq 1$ then the cofactor is a prime power and is divisible by $G$.
+
+### Computation
+The less performance-critical calculations in cofact use the GNU Multiple Precision (GMP) arithmetic library. However for the heavy lifting of modular squarings required by Pépin's test, the `gwnum` library is used since it is multi-threaded and therefore provides much higher performance. Even so, cofact (main branch) does not have Gerbitz error checking of calculations and does not save interim results to a file. So, for a Fermat cofactor test that is expected to run more than a few hours, it is preferable to first either generate the proof file using mprime / Prime95 (which has Gerbitz error checking and checkpoint / restart capability) or download the proof file from Catherine's [website](https://64ordle.au/fermat/). Once a Fermat number's proof file is in hand, `cofact -upr` can be used to test the new cofactor whenever a new factor of the Fermat number is discovered.
```

Comment:
Here, a couple of corrections need to be made to replace Gerbitz with Gerbicz.

In the paragraph below for Proof files – I added $F_{12}$ and $F_{13}$, so we now have the complete set of composite and incompletely factored Fermat numbers up as far as $F_{29}$. (Proof files are probably largely unnecessary below $F_{25}$ but given there’s only so many (18) from $F_{12}$ up to $F_{29}$ it’s probably better to have the whole set available.)